test/mobile/test_lite_script_module.py

```diff
@@ -55,13 +55,83 @@ def forward(self, x):
 
         input = torch.tensor([5])
         trace_module = torch.jit.trace(B(), input)
-        bytes = trace_module._save_to_buffer_for_lite_interpreter(_save_mobile_debug_info=True)
+        exported_module = trace_module._save_to_buffer_for_lite_interpreter(_save_mobile_debug_info=True)
```

Comment:
Update the variable name here from `bytes` to `exported_module`, because `bytes` is a built-in type in python.